config/prow/plugins.yaml

```diff
@@ -757,9 +757,11 @@ cherry_pick_unapproved:
 
     To merge this cherry pick, it must first be approved (`/lgtm` + `/approve`) by the relevant OWNERS.
 
-    **AFTER** it has been approved by code owners, please ping the **kubernetes/release-managers** team in a comment to request a cherry pick review.
+    **AFTER** it has been approved by code owners, please leave the following comment on a line **by itself, with no leading whitespace** to request cherry pick review from a Release Manager: **/cc kubernetes/release-managers**
 
-    (For details on the patch release process and schedule, see the [Patch Releases](https://k8s.io/releases/patch-releases) page.)
+    (This command should work for org members and non-org members alike.)
```

Comment:
Just a bit more info? replace line 762 with:

`(this bot command will request a review from release managers team - https://github.com/orgs/kubernetes/teams/release-managers and should work for all github users whether they are members of kubernetes github organization or not)`

Reply:
That link would not work for non org members anyways. I'm fine with this either way.